src/components/dashboard/FaceRecognition/FRError.js

```diff
@@ -56,7 +55,10 @@ const FRError = props => {
             paddingTop: 0,
           }}
         >
-          <Section.Title style={styles.mainTitle}> {`${getFirstWord(fullName)},\n${title}`}</Section.Title>
+          <Section.Title fontWeight="medium" textTransform="none">
+            {' '}
```

Comment:
If this space is intentional would be better on the line bellow I think

Reply:
I don't think that was intentional. Good catch!

Reply:
fixed